daffodil-japi/src/main/java/org/apache/daffodil/japi/package-info.java

```diff
@@ -257,7 +257,7 @@
  *  WritableByteChannel wbc = java.nio.channels.Channels.newChannel(os);
  *  DaffodilUnparseContentHandler unparseContentHandler = dp.newContentHandlerInstance(wbc);
  *  try {
- *   XMLReader xmlReader = SAXParserFactory.newInstance().newSAXParser().getXMLReader();
+ *   XMLReader xmlReader = SAXParserFactory().newSAXParser().getXMLReader;
```

Comment:
Don't you need the SAXParserFactory.newInstance() in Java? I don't think you can just do SAXParserFactory()?

Reply:
Ah, yes. That's a scala-ism that made it into the comments of the JAPI.